org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -619,6 +695,44 @@ package lang {
 		method apply(parameters...) native
 		method toString() native
 	}
+	
+	/**
+	 *
+	 * @author dodain
+	 * @since 1.4.5
+	 */	
```

Comment:
Just a random though about this: I am really convinced that the author tags from the java school are really meaningless.

They just talk about who created the class, but dont talk about who extended or mantained it. And all this information is actually in de source control.

Also, it doesnt actually help the user of your code. And it goes agains the shared code ownership ideal where all the devs are owners, mantainers and creators. 

Thus it doesn't fit well in oss projects, IMHO. 

If you want to give attribution, just put it in the readme, in a section for that. 

That said, I have a lot of time used such tags in the past, they are fun actually to write, and I hace notice this just in most recent years.  So, i am not saying we should remove them, but that we should discourage that práctica for new code.